introduction.md

```diff
@@ -1,2 +1,7 @@
 My name is Patrice Cadiot, i'm living in Paris, France 
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?